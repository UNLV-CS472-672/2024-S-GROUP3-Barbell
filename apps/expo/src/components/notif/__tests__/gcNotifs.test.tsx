import React from 'react'
import { render, screen } from '@testing-library/react-native'
import GcNotifs from '../gcNotifs'

test('RotatingBarbellIcon', async () => {
  render(<GcNotifs />)
  
  expect(screen.toJSON()).toMatchSnapshot()
})
