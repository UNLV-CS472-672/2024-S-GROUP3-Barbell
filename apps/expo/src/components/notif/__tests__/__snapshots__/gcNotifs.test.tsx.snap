// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RotatingBarbellIcon 1`] = `<View />`;
